benchmarks/performancecounters/apple_arm_events.h

```diff
@@ -966,11 +966,9 @@ struct AppleEvents {
   u64 counters_1[KPC_MAX_COUNTERS] = {0};
   static constexpr usize ev_count =
       sizeof(profile_events) / sizeof(profile_events[0]);
-
+  bool init = false;
+  bool worked = false;
```

Comment:
what is the reasoning behind this change?

Reply:
We have this...

```C++
struct AppleEvents {

  inline bool setup_performance_counters() {
    static bool init = false;
    static bool worked = false;
```

It is terrible code because if you create more than one AppleEvents, then your counters are broken. And we do.